src/components/Mobile/components/FilterMobile/FilterModal.tsx

```diff
@@ -90,6 +104,19 @@ const FilterModal = (props: FilterModalProps) => {
               : "All"}
           </Select>
         </Row>
+      ) : props.name === "Dates" ? (
+        <Row>
+          <p>{props.name}</p>
+          <Select select={isSelect}>
+            {filtersState.dateFrom && filtersState.dateTo
+              ? `${convertDateToDatePicker(
+                  filtersState.dateFrom
+                )} - ${convertDateToDatePicker(filtersState.dateTo)}`
+              : filtersState.dateFrom
+              ? convertDateToDatePicker(filtersState.dateFrom)
+              : "All"}
+          </Select>
+        </Row>
       ) : (
         <Row style={{ cursor: "auto", opacity: "0.5" }}>
```

Comment:
you can extend a Styled component like styled(Row) and move the inline-style there